PeerioTechnologies/peerio-desktop
# fix: don't mark messages as unread while idle

#### Relevant info and issue/PR links

https://app.clubhouse.io/peerio/story/17838/desktop-don-t-mark-new-messages-as-read-while-idle


#### Testing instructions

1. Wait for 5-6 minutes while doing nothing for computer to become idle.
2. Send message to the idle client from a different account and device.
3. Check that this message is not marked as read on the original client.
4. Move mouse or something like this on the original client to stop being idle.
5. Check that messages are marked as read.


----
### Repository owner


[PR guideline](https://github.com/PeerioTechnologies/peerio-icebear/blob/dev/docs/CONTRIBUTING.md)
